src/App.js

```diff
@@ -1,15 +1,42 @@
-import React from "react";
+import React, { useState, useEffect} from "react";
+import axios from 'axios';
+import Title from "./Components/Title";
+import Information from "./Components/Information";
+import Video from "./Components/Video";
+
 import "./App.css";
 
-function App() {
+const App = () => {
+  const [state, setState] = useState({
+    
+    date: '',
```

Comment:
creating these here, actually makes it pointless, cause they get replaced with the `res.data` that is coming from your axios call that you made